### PR TITLE
HTML-CSS-Popup

### PR DESCRIPTION
**Pull request by Levi123**
### HTML і CSS практика: Hooli-style Popup

[Demo](https://levi123.github.io/HTML-CSS-Popup-icons) |
[Code base](https://github.com/Levi123/html-css-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by HannaSyn** [comment]
Please, check navigation using keyboard (tab and space) and fix it. 
Now impossible to open menu with keyboard.

File: submissions/Levi123/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class = "menu">
+        <ul class = "navigationList">
+            <li class = "elementList"><a href="#">Mail</a></li>
```
Comment (HannaSyn) [inline submissions/Levi123/html-css-popup/index.html:13]:
```suggestion
            <li class="elementList"><a href="#">Mail</a></li>
```
You don't need spaces here

Comment (HannaSyn) [reply submissions/Levi123/html-css-popup/index.html:13]: Also please, read this [article](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/) and use BEM for class names

File: submissions/Levi123/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            <li class = "elementList"><a href="#">Images</a></li>
+            <li class = "elementList">
+                <input type="checkbox" id="popUp">
+                <label for="popUp" class="popUpBtn">
+                    <img src="icons/popup-button.png" alt = "pop-up menu icon"/>
```
Comment (HannaSyn) [inline submissions/Levi123/html-css-popup/index.html:18]: For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

File: submissions/Levi123/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                </label>
+
+                <div class="popUpBody">
+                    <ul class="popUpElements">
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/calendar.png" alt = "calendar" class = "imgMenu"/></a>
+                            <div>Calendar</div>
```
Comment (HannaSyn) [inline submissions/Levi123/html-css-popup/index.html:25]: Please, use `span` or `p` for strings

File: submissions/Levi123/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+                            <div>Apps</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/documents.png" alt = "documents" class = "imgMenu"/></a>
+                            <div>Documents</div>
+                        </li>
+                    </ul>
+                    <ul class = "popUpMoreElements">
+                        <input type="checkbox" id="popUpMore">
+                        <label for="popUpMore" class="popUpBtnMore">More</label>
```
Comment (HannaSyn) [inline submissions/Levi123/html-css-popup/index.html:74]: More button should have `:hover` and `:focus` states

File: submissions/Levi123/html-css-popup/index.html (hunk 2 above)
Comment (HannaSyn) [reply submissions/Levi123/html-css-popup/index.html:18]: Also, Menu button should have `:hover` and `:focus` states

File: submissions/Levi123/html-css-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+                            </ul>
+                        </div>    
+                    </ul>
+                </div>
+            </li>
+            <li class = "elementList"><a href="#">Profile</a></li>
+        </ul>
+    </nav>
+</body>
+</html>
```
Comment (HannaSyn) [inline submissions/Levi123/html-css-popup/index.html:111]: Please, always add empty line in the end of files

File: submissions/Levi123/html-css-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (HannaSyn) [inline submissions/Levi123/html-css-popup/index.html:7]: Please, add title of your page here

File: submissions/Levi123/html-css-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class = "menu">
```
Comment (HannaSyn) [inline submissions/Levi123/html-css-popup/index.html:11]: Please, wrap navigation by tag `header`

**Review by HannaSyn — CHANGES_REQUESTED** [review]
Nice work
Please, improve your code according my comments

File: submissions/Levi123/html-css-popup/index.html (hunk 1 above)
Comment (Levi123) [reply submissions/Levi123/html-css-popup/index.html:13]: Thank you, i real article, but further study is required

File: submissions/Levi123/html-css-popup/index.html (hunk 2 above)
Comment (Levi123) [reply submissions/Levi123/html-css-popup/index.html:18]: for icons left empty attributes alt and add :hover and :focus states

File: submissions/Levi123/html-css-popup/index.html (hunk 3 above)
Comment (Levi123) [reply submissions/Levi123/html-css-popup/index.html:25]: chande <div> to <p>

File: submissions/Levi123/html-css-popup/index.html (hunk 4 above)
Comment (Levi123) [reply submissions/Levi123/html-css-popup/index.html:74]: Add hover and focus to more button

File: submissions/Levi123/html-css-popup/index.html (hunk 5 above)
Comment (Levi123) [reply submissions/Levi123/html-css-popup/index.html:111]: add empty line for css and html files

File: submissions/Levi123/html-css-popup/index.html (hunk 6 above)
Comment (Levi123) [reply submissions/Levi123/html-css-popup/index.html:7]: add title to my page

File: submissions/Levi123/html-css-popup/index.html (hunk 7 above)
Comment (Levi123) [reply submissions/Levi123/html-css-popup/index.html:11]: wrap navigation to header tag

**Comment by HannaSyn** [comment]
Please, remove images from this pr

**Comment by Levi123** [comment]
> images

Sorry, images removed

**Review by Levi123 — COMMENTED** [review]
New version

**Review by Levi123 — COMMENTED** [review]
Delete all icons files

**Comment by Levi123** [comment]
Hello, images deleted

вт, 16 авг. 2022 г. в 12:22, Hanna Syniavska ***@***.***>:

**Comment by HannaSyn** [comment]
@Levi123 you didn't push changes, I don't see any, check Files changed tab

**Comment by HannaSyn** [comment]
Good work, please fix some small styles problems: 
1. focus looks weird, you can fix default focus or remove your custom and leave default
2. More button should have :focus state when you navigate with keyboard  
Screenshot 2022-08-18 at 11 21 12

**Comment by Levi123** [comment]
> Good work, please fix some small styles problems:
> 
> 1. focus looks weird, you can fix default focus or remove your custom and leave default
> 2. More button should have :focus state when you navigate with keyboard
> 
> Screenshot 2022-08-18 at 11 21 12

I fix focus state for all elements.

**Comment by HannaSyn** [comment]
Well done!

**Pull request merged**